### ISSUE TITLE
Xbox protocol 

### ISSUE THREAD

**Issue by 0xf3f** [NimVek/pipboy#1]
The android app includes classes from '/com/microsoft/xbox/smartglass/'.

**Comment by NimVek** [comment]
You have some documantaion about smartgalss protocol?

**Comment by 0xf3f** [comment]
No, sry. First time i have ever heard about it. My plan was to decompile the app to get some hints about the protocol. Its a unity app and i think the protocol code (PS4/PC) is not implemented in java.

You can decompile the smartclass classes, though. (I think its is illegal where I live, and microsoft will not like it if the protocol is here on github)

**Comment by luckydonald** [comment]
@joelday is also working on this: [RobCoIndustries/pipboy on gitter](https://gitter.im/RobCoIndustries/pipboy?at=5688553f653b30761d770787)

**Comment by joelday** [comment]
@NimVek @luckydonald See https://github.com/joelday/DarkId.SmartGlass for protocol reference. Haven't documented it yet. I'm currently finishing up work on the title channel support, so once that is done, you'll have an end-to-end example of how to do it.
